routing.go

```diff
@@ -27,6 +27,8 @@ import (
 // results will wait for the channel to drain.
 var asyncQueryBuffer = 10
 
+var putValueRepublishInterval = 6 * time.Hour
```

Comment:
Will change after the answer to the question [here](https://github.com/libp2p/go-libp2p-kad-dht/issues/323#issuecomment-523049043)